src/react-components/avatar-preview.js

```diff
@@ -63,6 +63,26 @@ function fitBoxInFrustum(camera, box, center, margin = DEFAULT_MARGIN) {
   camera.lookAt(center);
 }
 
+function getThemeBackground() {
+  const currentTheme = APP.store.state.preferences.theme;
+  const themes = window.APP_CONFIG?.theme.themes;
```

Comment:
The `?.` here implies that APP_CONFIG could be undefined, which means the `themes` variable could be undefined here too.